plugins/search-backend/src/schema/openapi.yaml

```diff
@@ -0,0 +1,132 @@
+openapi: 3.0.3
+
+info:
+  title: '@backstage/plugin-search-backend'
+  version: '1'
+  description: The Backstage backend plugin that provides search functionality.
+  license:
+    name: Apache-2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  contact: {}
+
+servers:
+  - url: /
+
+components:
+  examples: {}
+  headers: {}
+  parameters: {}
+  requestBodies: {}
+  responses: {}
+  schemas:
+    JsonObject:
+      type: object
+      properties: {}
+      additionalProperties:
+        type: string
```

Comment:
I just want to say that this in general isn't true for JsonObject - anything json serializable can go as values including arrays and null. So leaving this entirely unspecified/open should be OK right?

Reply:
Thanks for pointing that out, I'd meant to call that out initially. The existing zod schema was resolving recursively n times, which isn't possible to resolve with the current json-schema-to-ts library we're using. It also causes a fairly significant performance issue when resolving that recursive type even if we can't resolve it. I toyed around with using a 2 or 3 level deep resolution, but I think it makes more sense to just leave the schema fairly open here and keep the zod enforcement as is. Will add a comment here to encapsulate this.

Reply:
Yeah is it possible to even do `additionalProperties: {}`? I mean this data is by its very nature going to be JSON serializable anyway.